src/tree_data.h

```diff
@@ -997,6 +997,21 @@ struct lyd_node_opaq {
     const struct ly_ctx *ctx;       /**< libyang context */
 };
 
+/**
+ * @brief Structure of leafref links record.
+ */
+struct lyd_leafref_links_rec {
+    const struct lyd_node_term *node;           /** pointer to the data node itself */
+    const struct lyd_node_term **leafref_nodes; /** list of the leafref pointing to this data node [sized array](@ref sizedarrays)),
+                                                    By default it is empty. It is filled automatically by validation
```

Comment:
Probably good to mention that this is true only for `require-instance true` leafrefs.

Reply:
Done